src/pygieons/ecosystem_pkgs.py

```diff
@@ -41,7 +41,7 @@
         "vector": ['geopy', 'h3', 'osmnet', 'osmnx', 'pyntcloud', 'pyrosm', 'snkit'],
         "raster": ['earthengine-api', 'easystac', 'eemont', 'lidar', 'odc-stac', 'planetary-computer', 'pymap3d',
                    'pyotb', 'pyrosar', 'pystac', 'pystac-client', 'radiant-mlhub', 'rio-hist', 'rio-mucho', 'rio-tiler',
-                   'salem', 'satpy', 'sentinelsat', 'stackstac', 'verde', 'xarray-sentinel', 'xyzservices'],
+                   'salem', 'satpy', 'sentinelsat', 'stackstac', 'verde', 'xarray-sentinel', 'xyzservices', 'eoreader'],
```

Comment:
Could you add 'eoreader' after `"eemont"`, so the list would be in alphabetical order? 🙏🏻